modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/analysis/v0.1.0.json

```diff
@@ -0,0 +1,55 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/analysis/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:AnalysisShape",
+      "@type": "sh:NodeShape",
+      "label": "Model or simulation report analysis shape",
+      "comment": "The analysis strategy/protocol is defined in a protocol.",
+      "targetClass": "nsg:Analysis",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/shapes/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "name": "Model",
+              "description": "Used model.",
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/simulation/modelinstance/v0.1.0/shapes/ModelInstanceShape",
+              "class": "nsg:ModelInstance",
+              "minCount": 1
+            },
+            {
+              "path": "prov:used",
```

Comment:
The property shape prov:used is duplicated in this schema

Reply:
Two entities may be used: a Model and a VariableReport as far as I know. SO we need a qualifiedValueShape.